modules/model/shared/src/main/scala/observe/model/package.scala

```diff
@@ -54,26 +51,13 @@ extension (i: Instrument)
     // None
     case _                     => none
 
-// Resources come before Instruments
-given Order[Resource | Instrument] = Order.from:
```

Comment:
Is this because they are `Enumerated`?

Reply:
Yes, `Enumerated` provides an `Order` already.